index.sh

```diff
@@ -16,7 +19,7 @@ _() {
   GIT_AUTHOR_DATE="${YEAR}-01-01T18:00:00" \
     GIT_COMMITTER_DATE="${YEAR}-01-01T18:00:00" \
     git commit -m "${YEAR}"
-  git remote add origin "https://github.com/${USERNAME}/${YEAR}.git"
+  git remote add origin "https://${ACCESS_TOKEN}@github.com/${USERNAME}/${YEAR}.git"
```

Comment:
Can you share with me some references for this usage? I am not very sure if the protocol is https? Thanks

Reply:
+ https://stackoverflow.com/questions/68775869/support-for-password-authentication-was-removed-please-use-a-personal-access-to
+ https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#http-based-git-access-by-an-installation